rdkit/Dbase/DbUtils.py

```diff
@@ -435,19 +444,16 @@ def DatabaseToDatabase(fromDb, fromTbl, toDb, toTbl, fields='*', join='', where=
    FIX: at the moment this is a hack
 
   """
-  from io import StringIO
   sio = StringIO()
   sio.write(
     DatabaseToText(fromDb, fromTbl, fields=fields, join=join, where=where, user=user,
                    password=password))
-  sio.seek(-1)
+  sio.seek(0)
```

Comment:
I think this is likely correct, but it's definitely different from what was there before.